src/core_modules/capture-core/components/Pages/ViewEvent/TopBar.container.js

```diff
@@ -69,7 +67,8 @@ export const TopBar = ({
             isUserInteractionInProgress={isUserInteractionInProgress}
             formIsOpen={formIsOpen}
             onStartAgain={() => reset()}
-            isReadOnly={isReadOnly}
+            isReadOnlyOrgUnit
+            orgUnitTooltip
```

Comment:
Could we conditionally display the tooltip only when the page is in edit mode, like it is done when [editing a Tracker event](https://github.com/dhis2/capture-app/pull/4013/files#diff-01b9bc487c8b711adca97a557ba8b6141928d1a293b0c94a7640cd830808793bR69)? Thanks!